src/services/send-email.service.js

```diff
@@ -7,12 +7,11 @@ import JwtService from './jwt.service';
 class SendEmailService {
   /**
    * @param {email} email user email
-   * @param {token} token user token
    * @param {subject} emailSubject subject of the email
    * @param {body} emailBody description of the email
    * @returns {sendMessage} this function send an email
    */
-  static sendGridEmail(email, token, emailSubject, emailBody) {
```

Comment:
Is our features going to work as they used to, because this method was for all of us and now, you removed the token `arg`. Need explanation. Thanks

Reply:
the token is not used anywhere in the body. s

Reply:
Did you check that other features that were depending on this method are working. For example mine of user account email verification? The email sent should contain a token> Try with your email and see if it works then will be ok. Thanks